v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,280 @@
+
+If you encrypt a  message with a TLS public key, you know that only a holder of the matching private key will be able to decrypt it. And perhaps the holder of the public certificate identifies themself as your friend, your bank, your employer, or a government. But how can you trust that the certificate was ever actually held by the party you want to reach, rather than an imposter?
+
+This is a complex social and technological problem, and the primary solution used today by the internet as a whole, as well as organizations across the world, is mix of careful practices, social relationships, and digital technologies. It is what is known as Public Key Infrastructure (PKI).
+
+At its core, PKI is a hierarchy of cryptographically backed trust relationships between the following categories of interested party:
+
+- **Subscribers** wish to use the PKI to prove their identity to the world and offer secure services to others.
+- **Relying parties** wish to connect with subscribers, secure in the their knownledge of the identity of the latter.
+- **Certificate Authorities (CA)** are responsible for verifying the identity of subscribers (which can include subornidate certificate authorties).
+
+### Certificates, signing, trust and authority
+
+The core mechanism of PKI is a the PKI certificate, also known as a 'security certificate', 'digital certificate' or 'TLS certificate' (because it is used in TLS), or abbreviated "cert".
+
+A PKI certificate is a file containing the following:
+
+- A) A public key to be used for [key pair encryption, often including TLS](#what-is-transport-layer-security-tls).
+- B) Some metadata about:
+	- the party that allegedly holds the corresponding private key and who therefore is the only one capable of decrypting messages encrypted with **A**, most importantly at least one **name**, such as a domain name in a domain name registry system (DNS). The certifiate essentially functions as a badge or nametage, allowing the holder to claim to be the named party.
+	- the party signing the certificate, and the certificate authority (if any) that signed its public certificate
+- C) A list of actions the holder of the certificate is thereby authorized to perform.
+
+On its own, such a digital certificate is of no more value than a paper certificate. Perhaps less value, as it can be neither scribbled upon nor burned. However, digital certificates have the advantage that they can be cryptographically **signed**, again using the mechanism of a public/private key pair.
+
+**Signing** here means encrypting the certificate using a private key, so that it can be decrypted with a public key. So how does that help? This is the point where the boundaries between computing systems and social systems become very murky. By signing a certificate (and anyone with a private key can do this) a party is acting as a "certificate authority"; they are in effect asserting the validity of the identity claim being made by the certificate holder.
```

Comment:
```suggestion
**Signing** here means generating a hash of data (such as a certificate) that can be verified with the public key, using a function that is difficult to spoof without the corresponding private key. So how does that help? This is the point where the boundaries between computing systems and social systems become very murky. By signing a certificate (and anyone with a private key can do this) a party is acting as a "certificate authority"; they are in effect asserting the validity of the identity claim being made by the certificate holder.
```

This would be a good section on its own!